RecoHI/HiCentralityAlgos/python/HiHFFilterProducer_cfi.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+hiHFfilters = cms.EDProducer("HiHFFilterProducer",
```

Comment:
this should load hiHFFilterProducer_cfi created by the fillDescriptions and clone from it.
Also, the cfi should better match the name of the module defined inside: rename to hiHFfilters_cfi

Reply:
implemented in commit caafebe12b7b2161461ad101f5e52d7490bfa835